src/main/java/edu/hm/hafner/analysis/parser/CodeCheckerParser.java

```diff
@@ -21,7 +21,7 @@
 public class CodeCheckerParser extends LookaheadParser {
     private static final long serialVersionUID = -3015592762345283582L;
     private static final String CODE_CHECKER_DEFECT_PATTERN =
-            "^\\[(?<severity>CRITICAL|HIGH|MEDIUM|LOW)\\] (?<path>.+):(?<line>\\d+):(?<column>\\d+): (?<message>.*?) \\[(?<category>[^\\s]*?)\\]$";
+            "^\\[(?<severity>UNSPECIFIED|STYLE|CRITICAL|HIGH|MEDIUM|LOW)\\] (?<path>.+):(?<line>\\d+):(?<column>\\d+): (?<message>.*?) \\[(?<category>[^\\s]*?)\\]$";
```

Comment:
if you change this only, those lines will then counted as "LOW".
Is that what you intended?
@uhafner I don't even know if `Severity` provides additional levels which can be used?
Its the first time I look at this after it got merged. I need to refresh my knowledge about this project again.

Reply:
Yes, i think that would be fine. Not sure where else to place them.

Reply:
In the model there are only ERROR, HIGH, NORMAL and LOW available. So choose the severities which matches best in your case.

Reply:
Do you want me to explicitly list them. At the moment they use the fallback of LOW which is fine.

Reply:
It is not necessary to list all of them, the fallback is just fine.